go/install/terminate_mobile.go

```diff
@@ -0,0 +1,8 @@
+// +build ios android
+
+package install
+
+// TerminateApp will stop the Keybase (UI) app
```

Comment:
Maybe explain here why this is a no-op on mobile?

Reply:
copy pasta, will change